packages/next/src/server/web/spec-extension/unstable-cache.ts

```diff
@@ -38,7 +38,7 @@ async function cacheNewResult<T>(
       revalidate: typeof revalidate !== 'number' ? CACHE_ONE_YEAR : revalidate,
     },
     {
-      revalidate,
+      cacheControl: revalidate !== undefined ? { revalidate } : undefined,
```

Comment:
Does this not require an `expiry` time as well?

Reply:
It turned out that this was actually dead code. I've removed the property in #76500.